app/components/Modals/SendModal/ConfirmDetails/ConfirmDetails.jsx

```diff
@@ -0,0 +1,53 @@
+// @flow
+import React from 'react'
+
+import Button from '../../../Button'
+import GridIcon from '../../../../assets/icons/grid.svg'
+
+import baseStyles from '../SendModal.scss'
+import styles from './ConfirmDetails.scss'
+
+type Props = {
+  recipientData: Object,
+  confirmAndClose: () => any
+}
+
+export default class ConfirmDetails extends React.Component<Props> {
+  getRecipientData = (key: string) =>
+    this.props.recipientData[key] || 'Not specified'
```

Comment:
prefer lodash get over this syntax - if recipientData is not defined there will be an error get(this.props.recipientData, key)